Jenkinsfile

```
@@ -2,4 +2,11 @@
  See the documentation for more options:
  https://github.com/jenkins-infra/pipeline-library/
 */
-buildPlugin(useContainerAgent: true)
+buildPlugin(
+  forkCount: '1C', // Run a JVM per core in tests
+  // we use Docker for containerized tests
+  useContainerAgent: false,
```

Since the prior configuration used container agents, shouldn't the new configuration use container agents as well?
```suggestion
  useContainerAgent: true,
```

---

Since we currently have an outage with Windows container agents, I ran the job with `useContainerAgent: false`, though long term I think that we want `useContainerAgent: true`